tensorflow_addons/losses/triplet.py

```diff
@@ -77,30 +77,21 @@ def triplet_semihard_loss(
     """Computes the triplet loss with semi-hard negative mining.
 
     Args:
-      y_true: 1-D integer `Tensor` with shape [batch_size] of
+      y_true: 1-D integer `Tensor` with shape `[batch_size]` of
         multiclass integer labels.
       y_pred: 2-D float `Tensor` of embedding vectors. Embeddings should
         be l2 normalized.
       margin: Float, margin term in the loss definition.
-      distance_metric: str or function, determines distance metric:
-                       "L2" for l2-norm distance
-                       "squared-L2" for squared l2-norm distance
-                       "angular" for cosine similarity
-                        A custom function returning a 2d adjacency
-                          matrix of a chosen distance metric can
-                          also be passed here. e.g.
-
-                          def custom_distance(batch):
```

Comment:
would make sense to format this example as python example, or?

Reply:
seems that is impossible to make a code block under listing env

Reply:
ah, we could move it from params section and just move to general doc string examples